src/deposit/entities/deposit.entity.ts

```diff
@@ -1 +1,24 @@
-export class Deposit {}
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import * as mongoose from 'mongoose';
+
+@Schema()
+export class Deposit {
+  @Prop({
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  })
+  userId: string;
```

Comment:
tutaj trzeba pobrać shcemat Usera z `user/entities/user.entity`  i wtedy zamiast stringa jest  `User` (możesz zobaczyć u mnie w role jak to jest zrobione)